Lib/idlelib/idle_test/test_config.py

```diff
@@ -26,6 +29,154 @@ def tearDownModule():
     idleConf.userCfg = usercfg
 
 
+class IdleConfParserTest(unittest.TestCase):
+    """Test IdleConfParser works"""
+
+    config = """
+        [one]
+        one = false
+        two = true
+        three = 10
+
+        [two]
+        one = a string
+        two = true
+        three = false
+    """
+
+    def test_get(self):
+        parser = config.IdleConfParser('')
+        parser.read_string(self.config)
+
+        self.assertEqual(parser.Get('one', 'one', type='bool'), False)
+        self.assertEqual(parser.Get('one', 'two', type='bool'), True)
+        self.assertEqual(parser.Get('one', 'three', type='int'), 10)
+        self.assertEqual(parser.Get('two', 'one'), 'a string')
+        self.assertEqual(parser.Get('two', 'two', type='bool'), True)
+        self.assertEqual(parser.Get('two', 'three', type='bool'), False)
+        self.assertEqual(parser.Get('two', 'two'), 'true')
+        self.assertEqual(parser.Get('two', 'three'), 'false')
+        self.assertEqual(parser.Get('not', 'exist'), None)
+        self.assertEqual(parser.Get('not', 'exist', default='DEFAULT'), 'DEFAULT')
```

Comment:
This could be done with a subtest with 5 cases and a subtest with 4 cases and the remaining test.  But I think it not worthwhile here.  What I have done in situations like this and would like to do here is to name the bound method, "equal = self.assertEqual", and then use that:  "equal(parser.Get(...., False)".  It is then easier to focus on what is being compared.